src/computation/logic/utils.ts

```diff
@@ -185,7 +186,7 @@ export function distributeLevelingSeatsOnDistricts(
         if (levelingSeats.length === 0) {
             finishedDistricts = [];
             levelingSeats = generateLevelingSeatArray(
-                payload.algorithm,
+                AlgorithmType.SAINTE_LAGUE,
```

Comment:
Are leveling seat arrays always SAINTE_LAGUE?

Reply:
Bernt said we could always use Sainte Laguës for distribution leveling seats on each district.

Reply:
The whole function should really be rewritten.

Reply:
One wonderful day when there is time!